mon-pix/app/templates/components/navbar-burger-menu.hbs

```diff
@@ -18,8 +18,13 @@
   </LinkTo>
 </div>
 
-{{#unless isInRouteWithoutLinksInHeader}}
+{{#unless this.isInRouteWithoutLinksInHeader}}
```

Comment:
**question (non-blocking):** Pas besoin de `this` pour les `currentUser` en lignes 2, 5 et 8 ?